### PR TITLE
12300 Filing UI - update logic for displaying compliance warnings according to BE changes

### PR DESCRIPTION
**Pull request by lewischenstudio**
*Issue #:* [/bcgov/entity#12300](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/12300)

*Description of changes:*
- Changed `complianceWarnings` to `warnings`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Review by vysakh-menon-aot — APPROVED** [review]

**Review by severinbeauvais — COMMENTED** [review]
Looks good so far.

File: src/interfaces/business-interface.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { EntityState, CorpTypeCd } from '@/enums'
 import { IsoDatePacific, ApiDateTimeUtc } from '@bcrs-shared-components/interfaces'
 
 /** The Compliance Warning object. */
-export interface ComplianceWarning {
+export interface Warning {
```
[the remark this answers is not in the document]
Comment (severinbeauvais) [reply src/interfaces/business-interface.ts]: Awesome, thanks!

File: src/interfaces/state-interface.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export interface StateIF {
   adminFreeze: boolean
   identifier: string
   businessNumber: string
-  complianceWarnings: Array<ComplianceWarning>
+  warnings: Array<BusinessWarningIF>
```
Comment (severinbeauvais) [inline src/interfaces/state-interface.ts:23]: Should this be businessWarnings too?

**Review by severinbeauvais — APPROVED** [review]
Thanks for the changes. Looks great!

**Pull request merged**